src/rust/engine/fs/store/src/lib.rs

```diff
@@ -941,6 +949,7 @@ impl Store {
       .await?
       .walk(&mut |_, entry| match entry {
         directory::Entry::File(f) => file_digests.push(f.digest()),
+        directory::Entry::Symlink(_) => todo!(),
         directory::Entry::Directory(_) => (),
```

Comment:
Nothing to do here... if the symlink is relative within the tree, then it will already be visited: if it is outside the tree then there is nothing to do.

```suggestion
        directory::Entry::Symlink(_) | directory::Entry::Directory(_) => (),
```